static/sass/partials/_content.scss

```diff
@@ -0,0 +1,47 @@
+.content{
+  padding-top: 40px;
+}
+.content-wrapper{
+  background-color: $secondary-bg;
+  .content-wrapper2{
+    h2{
+      text-align: center;
+      margin: 50px;
+      margin-bottom: 15px;
+      font-weight: bolder;
+    }
+
+    p{
+      text-align: center;
+    }
+
+   .button{
+      text-align: center;
+      .sponsor-button{
```

Comment:
Please make this generic. This will be a link, you can call the class "link-btn" and should be able to use it anywhere in the site (not just content). Probably this needs to be moved to base.scss?

Reply:
Done.